### DISCUSSION TITLE
rnaSPAdes hybrid assembly with YAML file

### DISCUSSION THREAD

**Discussion by poecile-palustris** [ablab/spades#934] [Q&A] [answered]
Good morning,

I would like to make a hybrid assembly (Illumina + Oxford Nanopore) using rnaSPAdes. I have a lot of libraries, so I have to set them in a YAML file. Do I understand correctly that in this case I need to specify both those and other libraries in YAML, and write the code as follows:

`rnaspades.py --dataset libraries.yaml --nanopore -o OUTPUT`
(without specifying anything after `--nanopore`)

with YAML like this:

 ```
[
      {
        orientation: "fr",
        type: "paired-end",
        right reads: [
          "/FULL_PATH_TO_DATASET/lib_pe1_right_1.fastq",
          "/FULL_PATH_TO_DATASET/lib_pe1_right_2.fastq" 
        ],
        left reads: [
          "/FULL_PATH_TO_DATASET/lib_pe1_left_1.fastq",
          "/FULL_PATH_TO_DATASET/lib_pe1_left_2.fastq" 
        ]
      },
      {
        type: "nanopore",
        single reads: [
          "/FULL_PATH_TO_DATASET/nanopore_1.fastq" 
          "/FULL_PATH_TO_DATASET/nanopore_2.fastq" 
        ]
      }
 ]
```
Thank you in advance!

Best regards, 
Poecile

**Comment by asl** [comment, marked as answer]
Hello

No need to specify `--nanopore` as you have already specified the type of the library in the YAML, so it is redundant.

**Reply by poecile-palustris** [reply]
Anton, thank you very much for the fast answer!
